posts/2025-07-03_swab-ww-comparison/index.qmd

```diff
@@ -0,0 +1,410 @@
+
+geom_means <- detection %>%
+  summarize(
+    ci_at_detection_ww = gm(ci_at_detection_ww),
+    ci_at_detection_swab = gm(ci_at_detection_swab),
+    ratio = ci_at_detection_ww / ci_at_detection_swab,
+    .groups = "drop",
+  )
+
+geom_means %>%
+  select(- group) %>%
+  rename(
+    wastewater = ci_at_detection_ww,
+    swab = ci_at_detection_swab,
+  ) %>%
+  kable(digits = c(NA, 5, 5, 2)) %>%
+  column_spec(c(2, 3, 4), width = "3cm", monospace = TRUE) 
+```
+
+We can also plot the full joint distributions of cumulative incidence at detection:
+
+```{r}
+#| label: joint-distribution
+#| fig-cap: "Cumulative incidence at detection: wastewater vs. swab. Black dot show geometric means. Grey line is equal incidence. Points above the grey line indicate earlier detection in swabs and vice versa."
+
+detection %>%
+  ggplot(aes(ci_at_detection_swab, ci_at_detection_ww, color = group)) +
+  geom_abline(intercept = 0, slope = 1, color = "grey") +
+  geom_density_2d() +
+  geom_point(data = geom_means, color = "black") +
+  scale_x_log10() +
+  scale_y_log10() +
+  facet_wrap(~species, nrow = 2) +
+  theme(legend.position = "none") +
+  labs(
+    x = "cumulative incidence at detection in swabs",
+    y = "cumulative incidence at detection in wastewater",
+    )
+```
+
+Finally, we can plot the distribution of the ratio of CI at detection in the two methods:
+
+```{r}
+#| label: ratio
+#| fig-cap: "**Ratio of cumulative incidence at detection in wastewater vs swabs **. Vertical bars are 15, 50, and 85th percentiles. Ratio < 1 means earlier detection in wastewater; ratio > 1 means earlier detection in swabs."
+
+detection %>%
+  mutate(ratio = ci_at_detection_ww / ci_at_detection_swab) %>%
+  ggplot(aes(ratio, species, fill = group)) +
+  geom_vline(xintercept = 1, color = "darkgrey") +
+  geom_violin(draw_quantiles = c(0.15, 0.5, 0.85)) +
+  scale_x_continuous(
+    transform = "log10",
+    labels = scientific,
+    breaks = c(1e-2, 1, 1e2)
+    ) +
+  theme(legend.position = "none") +
+  labs(x = NULL, y = NULL)
+```
+
+These simulations show that, *with these sampling and sequencing parameters*, for most of these viruses,
+we expect detection in swabs at 1.3 to 15 times lower cumulative incidence than in wastewater.
+The big exception is SARS-CoV-2, which sheds a lot in wastewater, for which the simulations suggest 10x lower relative abundance at detection in wastewater than in swabs.
+
+A major caveat is that we are not modeling the delay between sample collection and
+detection.
+With exponential growth, delay can be very costly in terms of cumulative incidence.
+With the `r doubling_time`-day doubling time assumed here, a two-day delay results in a `r round(exp(2 * growth_rate), 1)`-fold
+increase in the cumulative incidence at detection.
+
+# Heuristic analysis
```

Comment:
Some optional suggestions about this section:

It would be helpful to know which of the two regimes we are in for the various viruses.

I personally found it helpful to think of the two regimes for swabs as depending on whether the expected number of junction reads per infected swab, equal to `mu_swab * pr_junction_swab * total_reads_swab / swab_pool_size`, is greater or less than 1. One way to quickly see this is to divide out pool size from the min expression.